### PR TITLE
melody code frame testcase

### PR DESCRIPTION
**Pull request by ayusharma**
**Type**:  Test case

The following has been addressed in the PR:
- Test case for `melody-code-frame`

### CODE REVIEW

**Comment by ayusharma** [comment]
@efegurkan Could you please check the Travis CI or trigger the CI again. 

Getting following error in my last 2 PRs 😞 . Tests are failing only for Node 8.

```
[1/5] Validating package.json...
[2/5] Resolving packages...
[3/5] Fetching packages...
error An unexpected error occurred: "https://registry.yarnpkg.com/babel-core/-/babel-core-6.26.0.tgz: ETIMEDOUT".
info If you think this is a bug, please open a bug report with the information provided in "/home/travis/build/trivago/melody/yarn-error.log".
info Visit https://yarnpkg.com/en/docs/cli/install for documentation about this command.
The command "yarn" failed and exited with 1 during .
```

File: packages/melody-code-frame/__tests__/CodeFrameSpec.js [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+describe('codeframe', () => {
+    it('should match the codeframe output with line 2 and col 4', () => {
+        const input = {
+            rawLines,
+            lineNumber: 2,
+            colNumber: 4,
+            length: 1,
+        };
+
+        const output = `  1 | <div class="home">
```
Comment (efegurkan) [inline packages/melody-code-frame/__tests__/CodeFrameSpec.js]: You can use snapshots for that.

**Comment by ayusharma** [comment]
I'll update the PR with more edge cases with snapshot tests.

**Comment by ayusharma** [comment]
I think there should be a check for the `lineNumber`, ` colNumber` and `length` in
https://github.com/trivago/melody/blob/27745d7531aed047f1491c33b9306adb7df87993/packages/melody-code-frame/src/index.js#L21

A check could be 
```
if (lineNumber < 1 || colNumber < 1 || length < 0) {
        throw new Error("'lineNumber' & 'colNumber' should be greater than 0 and 'length' should be greater than -1")
}
```

**Comment by efegurkan** [comment]
> I think there should be a check for the lineNumber, colNumber and length in
melody/packages/melody-code-frame/src/index.js 

It is not necessary since we have `Math.max` and `Math.min` checks at [line 23-24](https://github.com/trivago/melody/blob/27745d7531aed047f1491c33b9306adb7df87993/packages/melody-code-frame/src/index.js#L23)
